### PR TITLE
change repo URL to the new one.

### PR DESCRIPTION
**Pull request by tiansiyuan**

### CODE REVIEW

File: examples/helmet_object_detection/notebook/README.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ You can refer to the [Notebook Document](https://elements-of-ai.github.io/kubefl
 - Clone this repo in the notebook and install the dependencies in the requirements.txt:
 
 ```bash
-$ !git clone https://github.com/elements-of-ai/kubeflow-docs.git
-$ cd examples/helmet_object_detection/notebook
+$ git clone https://github.com/vmware/vSphere-machine-learning-extension.git
```
Comment (liuqi) [inline examples/helmet_object_detection/notebook/README.md:19]: Exclamation mark is removed. That is a on-purpose change? @harperjuanl could you please help to confirm? Thanks.

Comment (liuqi) [reply examples/helmet_object_detection/notebook/README.md:19]: Also, there is old link in line 7. Could you please help to change together? Thanks.

Comment (harperjuanl) [reply examples/helmet_object_detection/notebook/README.md:19]: Exclamation mark should be used when executed in the jupyter notebook (.ipynb), the same as the following commands

Comment (tiansiyuan) [reply examples/helmet_object_detection/notebook/README.md]: > Also, there is old link in line 7. Could you please help to change together? Thanks.

Done.

The $ before each command is also removed.

**Review by liuqi — APPROVED** [review]

**Pull request closed without merging**